source/amazon/services/troubleshooting.rst

```diff
@@ -8,46 +8,232 @@ Troubleshooting
+
+- **Debug level 2**: This is the highest level of verbosity. Every message type is dump into the ``ossec.log`` file, including ``DEBUG`` messages which contain the details of the different operations performed by the module. This is the recommended mode when troubleshooting the module.
+
+
+Follow these steps to enable debug mode:
+
+#. Add the following line to the ``WAZUH_PATH/etc/local_internal_options.conf`` file, specifying the desired debug level:
+
+    .. code-block:: none
+
+        wazuh_modules.debug=2
+
+
+#. Restart the Wazuh service
+
+
+.. tabs::
+
+  .. group-tab:: Wazuh manager
+
+      .. tabs::
+
+        .. tab:: For Systemd
+
+          .. code-block:: console
+
+            systemctl restart wazuh-manager
+
+        .. tab:: For SysV Init
+
+          .. code-block:: console
+
+            service wazuh-manager restart
+
+  .. group-tab:: Wazuh agent
+
+      .. tabs::
+
+        .. tab:: For Systemd
+
+          .. code-block:: console
+
+            systemctl restart wazuh-agent
+
+        .. tab:: For SysV Init
 
-    When an error occurs when trying to collect and parse logs for an AWS service, the ``ossec.log`` will output an error such as below:
+          .. code-block:: console
+
+            service wazuh-agent restart
+
+
+.. Note::
+        Don't forget to disable debug mode once finished. Leaving debug mode enabled could result in the addition of large amounts of logs in the ``ossec.log`` file.
+
+.. _aws_events_processed:
+
+Checking if logs are being processed
+------------------------------------
+
+The easiest way to check if the logs are being processed, regardless of the type of bucket or service configured and regardless of whether alerts are being generated or not is by using the :ref:`reference_ossec_global_logall_json` parameter.
+
+To understand how the :ref:`reference_ossec_global_logall_json` parameter works it is necessary to learn about the flow that is followed when processing a log until the corresponding alert is displayed in the Wazuh UI. It is as follows:
+
+#. The module downloads the logs available in AWS for the requested date and path. Check the :ref:`amazon_considerations` page to learn more about how to properly filter the logs.
+#. The content of these logs is sent to the analysis engine in the form of an ``Event``.
+#. The analysis engine evaluates these events and compares them with the different rules available. If the event matches any of the rules an alert is generated, which is what ultimately is shown in the Wazuh UI.
+
+With this in mind, it is possible to make use of the :ref:`reference_ossec_global_logall_json`. When this option is activated, Wazuh stores into ``WAZUH_PATH/logs/archives/archives.json`` file every event sent to the analysis engine whether they tripped a rule or not. By checking this file it is possible to determine if the AWS events are being sent to the analysis engine and therefore working properly.
+
```

Comment:
Add a note telling that it is important to disable the `logall_json` parameter once the debugging has finished.
```suggestion

.. warning::
It is important to disable the `logall_json` parameter once the troubleshooting session has ended, having that option enabled for a long time could cause a high disk space consumption.
```

Reply:
Done with a note instead of warning